### ISSUE TITLE
[BUG] AWS config file is not being read correctly anymore

### ISSUE THREAD

**Issue by hrzbrg** [aws-samples/amazon-bedrock-client-for-mac#54]
**Describe the bug**
Since Version 1.2.3 it seems like the client does not read my `~/.aws/config` file properly anymore. I get the following error:

![Screenshot 2024-11-28 at 09 09 19](https://github.com/user-attachments/assets/7c1aa6f8-3be0-4d92-8b41-d8fe6d0b3773)

So it looks like the file is parsed and I can select the available account profiles, but the error always pops up. Excerpt from my `~/.aws/config`

```
[profile dev]
sso_start_url = https://sflx.awsapps.com/start
sso_region = eu-central-1
sso_account_id = <redacted>
sso_role_name = sflx_administrator
region = eu-central-1
```

**Environment (required):**
 - macOS version: 14.7
 - Amazon Bedrock Client for Mac version: 1.2.3 and 1.2.4

**Comment by NathanHazout** [comment]
I can confirm similar behavior. I wanted to create a new profile, which I added both in config and credentials, but the app kept looking at "default" no matter what I picked in the dropdown.

I ended up putting my bedrock credentials in "default" instead.

**Comment by didhd** [comment]
We are taking this issue very seriously and will fix it promptly. We apologize for any inconvenience caused.

**Comment by didhd** [comment]
@NathanHazout Fixed in v1.2.5
